mister.py

```diff
@@ -105,6 +108,18 @@ def handleRunBat(gGator):
         gGator.logger.log('    <ERROR> run.bat not found', gGator.logger.ERROR)
 
 
+# Convert sound command for MiSTeR
+def convertSoundConfig(line):
+    if 'mididevice' in line:
+        if 'mt32' in line:
+            return 'mt32-pi -m -v'
+        elif 'fluidsynth' in line:
+            return 'mt32-pi -g -v'
+        elif 'default' in line:
+            return 'mt32-pi -g -v'
```

Comment:
as you return the same thing the two for fluidsynth and default you can use an or :
`elif 'fluidsynth' in line or 'default' in line :`

Reply:
This is basically the same reasoning as above. Except that I would agree that the version using "or" is indeed a bit more readable. But since I am pretty sure that this mapping might change in the future I divided it into two cases.

Same disclaimer as above. If you want this changed, I'll do so.